pghoard/basebackup.py

```diff
@@ -718,11 +718,11 @@ def create_and_upload_chunks(
                     # and this assumption greatly simplifies the logic.
                     task_to_wait = pending_compress_and_encrypt_tasks.pop(0)
                     chunk_files.append(task_to_wait.result())
-                    self.metrics.gauge(
-                        "pghoard.basebackup_estimated_progress",
-                        float(len(chunk_files) * 100 / len(chunks)),
-                        tags={"site": self.site}
-                    )
+                self.metrics.gauge(
+                    "pghoard.basebackup_estimated_progress",
+                    float(len(chunk_files) * 100 / len(chunks)),
+                    tags={"site": self.site}
+                )
```

Comment:
`chunks` doesn't seem to be modified and the metrics was already sent every time `chunk_files` was modified.

I could be missing something but it seems the proposed changed wouldn't have any effect ?